src/db/models/db_ledger_entry.rs

```diff
@@ -24,6 +26,12 @@ pub struct DbLedgerEntry {
     pub timestamp: PgBigIntU32,
 }
 
+impl fmt::Display for DbLedgerEntry {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.rune_id)
+    }
+}
```

Comment:
was this for testing? otherwise we should either print all fields here or just remove this `impl`